devtools-ci-index.yaml

```diff
@@ -1520,11 +1520,17 @@
 - rh-che-automation-template: &rh-che-automation-template
     name: 'rh-che-automation-template'
     concurrent: false
+    wrappers:
+        - quay_credentials
+    <<: *job_template_defaults
+
+- rh-che-automation-depbuild-template: &rh-che-automation-depbuild-template
+    name: 'rh-che-automation-depbuild-template'
+    concurrent: false
```

Comment:
Is this really what you want to do? Removing all the wrappers except for one from `rh-che-automation-template` and having all those wrappers in `rh-che-automation-depbuild-template`? I would expect, that depbuild would have only quay.io creds (to push the image)...

Reply:
Thanks for the catch!